apps/web/src/components/common/Table/DataRow.tsx

```diff
@@ -12,7 +13,7 @@ export const DataRow = ({ datatestid, title, children }: DataRowProps): ReactEle
 
   return (
     <FieldsGrid data-testid={datatestid} title={title}>
-      {children}
+      <Typography variant="body1">{children}</Typography>
```

Comment:
@usame-algan is this needed for Earn or is it a leftover from a merge?

Reply:
No, these are used in the confirmation views for the labels and we decided to update them to make them bigger consistently. Before they were only 14px and of different sizes but now they should all be 16px.

Reply:
AFAIR in Signing UX (on dev), they are back to 14px.

Reply:
We might need to use `DataRow` more widely in other flows as well. Afaik it is currently only used for swaps confirmation and earn now.